AutoCollection/Requests.ts

```diff
@@ -67,53 +67,57 @@ class AutoCollectRequests {
             });
 	    }
     }
-
+    
     /**
-     * Tracks a request
+     * Tracks a request synchronously (doesn't wait for response 'finish' event)
      */
-    public static trackRequest(client:Client, request:http.ServerRequest, response:http.ServerResponse, properties?:{ [key: string]: string; }) {
+    public static trackRequestSync(client: Client, request: http.ServerRequest, response:http.ServerResponse, ellapsedMilliseconds?: number, properties?:{ [key: string]: string; }, error?: any) {
```

Comment:
I modeled this after the node file system api, where it's sort of opposite from C#. For example fs.writefile writes a file asynchronously, and fs.writefilesync writes it synchronously. Also this way I don't have to make a breaking API change by renaming trackRequest to trackRequestAsync